### DISCUSSION TITLE
安装完小雅alist以后无法使用管理后台

### DISCUSSION THREAD

**Discussion by hujeff** [AlistGo/alist#7290] [Q&A]
安装环境
群辉7.2.1-69057
Container Manager20.10.23-1437

使用"bash -c "$(curl --insecure -fsSL https://ddsrem.com/xiaoya_install.sh)""命令的第1项进行安装，安装过程中全部使用默认设置，没有提示错误，安装完成后使用IP:5678能正常访问。
![image](https://github.com/user-attachments/assets/a15ea7ec-cab8-4cb0-9635-f9d07b406990)

使用IP:5678/@manage无法访问后台，提示"403 Forbidden"。
![image](https://github.com/user-attachments/assets/3a3f4554-14a7-44c6-87e6-73e98cdfc3c7)

尝试过在guestpass.txt中添加账户和密码，重启容器，重启docker以后无效。
尝试过在根目录添加guestpass.txt，目的是设置为强制登录，但重启容器，重启docker以后并没有生效，使用IP:5678依然直接出现全部列表。
尝试使用命令"docker exec -it xiaoya ./alist admin set 123456"命令正常执行完成，但依然无法访问后台。
![image](https://github.com/user-attachments/assets/2b676996-07e7-4331-a8a3-e131b028c3eb)

我需要进入后台获取alist的token设置tvbox，请大佬们指点一二，万分感谢。

**Comment by hujeff** [comment]
请告诉我怎么强制登录，使用命令也好，添加文件也好，任何方法都行，只要能够登录，非常感谢。

**Comment by Simonzhang8** [comment]
小雅的网页后台是被禁用的。如果只是获取令牌可以参考以下方法：
```
docker exec -i xiaoya sqlite3 data/data.db <<EOF
select value from x_setting_items where key = "token";
EOF
```
另外，小雅的其他问题建议直接去小雅那边提问~

**Comment by hujeff** [comment]
实在不好意思，我在小雅那边真的没法提问，没有梯子，没有外网的手机，连他们的群都进不去……
